src/tools/rest-frontend/resources/structure.json

```diff
@@ -0,0 +1,1896 @@
+[
+    {
```

Comment:
Oh, now I got it. Sure, you can simply add not used elements, won't break anything. E.g. `dev-comment` would be good.